src/main/java/org/graylog/plugins/threatintel/adapters/spamhaus/SpamhausEDROPDataAdapter.java

```diff
@@ -184,9 +183,6 @@ public Descriptor() {
             @JsonProperty(TYPE_FIELD)
             public abstract SpamhausEDROPDataAdapter.Config.Builder type(String type);
 
-            @JsonProperty("refresh_interval")
-            public abstract SpamhausEDROPDataAdapter.Config.Builder refreshInterval(long refreshInterval);
```

Comment:
Wouldn't it make more sense to make this configurable via the Graylog REST API and the web interface instead of hard-coding it to 12 hours?

I do agree with a default of 12 hours, though. Ideally we enforce a lower limit of 1 hour.

https://www.spamhaus.org/faq/section/DROP%20FAQ#218 says:
> # How often should my system fetch the DROP / EDROP list?
> Please DO NOT auto-fetch the DROP / EDROP list more than once per hour!
> 
> The DROP list changes quite slowly. There is no need to update cached data more than once per hour, in fact once per day is more than enough in most cases. Automated downloads must be at least one hour apart. Excessive downloads may result in your IP being firewalled from the Spamhaus website.

Reply:
I do agreee, but I think this is way beyond the scope of this PR and should be implemented as a general feature of lookup tables instead.

Reply:
I just ran into a problem updating my development setup due to the `refresh_interval` removal and thus saw this.

The lookup table refresh system gets the refresh interval from a data adapter via `LookupDataAdapter#refreshInterval()`. The data adapter in Graylog core all have a configurable refresh interval. (except the HTTPJSONPath one where it doesn't make sense)

So what do you mean that this should be implemented as a general feature in lookup tables, @dennisoelkers? I think it already is. :smiley:

Reply:
Actually, I think this one should be reverted and the `refresh_interval` should be used in the `LookupDataAdapter#refreshInterval()` to avoid hardcoding it.